modules/javafx.controls/src/main/java/javafx/scene/control/skin/ToolBarSkin.java

```diff
@@ -588,7 +609,11 @@ private void addNodesToToolBar() {
                 break;
             }
         }
+        return hasOverflow;
+    }
 
+    private void organizeOverflow(double length, boolean hasOverflow) {
+        double x;
```

Comment:
This declaration can be moved to it's first use.

Reply:
I reorganized the methods close to its first use.

Reply:
Thanks.
My comment was for variable 'x' declaration. It is declared on line 578, but initialized on line 589.

Reply:
Done.